installation/sql/mysql/extensions.sql

```diff
@@ -882,3 +882,43 @@ CREATE TABLE IF NOT EXISTS `#__action_logs_users` (
   PRIMARY KEY (`user_id`),
   KEY `idx_notify` (`notify`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `description` text NOT NULL,
+  `title` varchar(255) NOT NULL,
+  `date` DATE NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `published` tinyint NOT NULL DEFAULT 1,
+  `ordering` int NOT NULL DEFAULT 0,
+  `created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+  `created_by` int unsigned NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ,
+  `modified_by` int unsigned NOT NULL DEFAULT 0,
+  `state` TINYINT NOT NULL DEFAULT '1'
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS `#__guidedtours` (
  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
  `description` text NOT NULL,
  `title` varchar(255) NOT NULL,
  `date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
  `published` tinyint NOT NULL DEFAULT 1,
  `ordering` int NOT NULL DEFAULT 0,
  `created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  `created_by` int unsigned NOT NULL DEFAULT 0,
  `modified` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
  `modified_by` int unsigned NOT NULL DEFAULT 0,
  `state` TINYINT NOT NULL DEFAULT '1'
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
```
Details see previous suggestions.